base/util.jl

```diff
@@ -689,12 +689,14 @@ to the standard libraries before running the tests.
 If `propagate_project` is true the current project is propagated to the test environment.
 If a seed is provided via the keyword argument, it is used to seed the
 global RNG in the context where the tests are run; otherwise the seed is chosen randomly.
+The argument `julia_args` can be used to pass custom julia arguments to the test process.
```

Comment:
This to me sounds like you're passing additional items that will populate `ARGS` rather than additional flags for the executable. Perhaps this would help clarify?:
```suggestion
The argument `julia_args` can be used to pass custom `julia` command line flags to the test process.
```

Reply:
Thanks, I agree